src/platform_impl/macos/view.rs

```diff
@@ -378,6 +389,9 @@ declare_class!(
 
         #[method(insertText:replacementRange:)]
         fn insert_text(&self, string: &NSObject, _replacement_range: NSRange) {
+            // TODO: We need some Nihonjin IME devs' help regarding how to use _replacement_range correctly.
+            // The "_replacement_range" is at least used in Japanese IMEs.
+            // Therefore, Nihonjin IME devs are more familiar with Japanese IMEs on mac.
```

Comment:
Just remove all this comment all together, there's no API to even utilize the `replacement_range`.

If you don't really want to do so, you can comment like `/// TODO: Requires surrounding text reporting on the Window.`.

Reply:
> there's no API to even utilize the replacement_range

Prejudices are not professional at all. This parameter is supported in Qt:
https://github.com/qt/qtbase/blob/3941fe697b9fea633f85321031ece765c532d46d/src/plugins/platforms/cocoa/qnsview_complextext.mm#L41-L93

Reply:
No API in **winit**.

Reply:
> No API in **winit**.

Then supply the API later.

Reply:
Yes, but there's no point in `TODO` in code when you can have all of that in the proper issues. No one really looks in the code TODOs. The only exception I'd say use that if it's for yourself, which is not the case mostly with most TODOs.